test/nightly/run_combined_tests.py

```diff
@@ -0,0 +1,464 @@
+"""Generic metrics runner for running performance + accuracy on models.
+
+This provides a simple way to run both performance and accuracy tests on a list of models,
+using the same abstractions as the existing nightly tests:
+- NightlyBenchmarkRunner for performance
+- run_eval (SimpleNamespace) for accuracy
+
+Hardware is determined by GitHub Actions jobs, not by this layer.
+
+Usage:
+    from nightly_metrics import run_metrics
+    from sglang.test.test_utils import ModelLaunchSettings
+
+    models = [
+        ModelLaunchSettings("meta-llama/Llama-3.1-8B-Instruct", tp_size=1),
+        ModelLaunchSettings("Qwen/Qwen2-57B-A14B-Instruct", tp_size=2),
+    ]
+
+    # Run both perf + accuracy for all models
+    result = run_metrics(
+        models=models,
+        run_perf=True,
+        run_accuracy=True,
+        is_vlm=False,
+    )
+"""
+
+import gc
+import time
+from types import SimpleNamespace
+from typing import List, Optional, Tuple
+
+from nightly_utils import NightlyBenchmarkRunner
+
+from sglang.srt.utils import kill_process_tree
+from sglang.test.run_eval import run_eval
+
+try:
+    import torch
+except ImportError:
+    torch = None
+from sglang.test.test_utils import (
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    ModelLaunchSettings,
+    _parse_int_list_env,
+    popen_launch_server,
+)
+
+
+def cleanup_between_tests(delay_seconds: int = 10) -> None:
```

Comment:
Do we actually need this part? The regular test server launch just use kill process tree.